# Set EGM mode
int64 mode
---
bool success
string msg
